### DISCUSSION TITLE
Generate QR

### DISCUSSION THREAD

**Discussion by maderaiindraperkasa** [ikhsan3adi/absensi-sekolah-qr-code#52] [Q&A] [answered]
Gan, kenapa tidak mau generate QR ya? padahal depedences seperti [Myth Auth Library](https://github.com/lonnieezell/myth-auth)
[Endroid QR Code Generator](https://github.com/endroid/qr-code)
[ZXing JS QR Code Scanner](https://github.com/zxing-js/library) sudah saya install menggunakan composer

terimakasih sebelumnya gan

**Comment by ikhsan3adi** [comment]
Apakah php extension `-gd` sudah diaktifkan?

**Reply by maderaiindraperkasa** [reply]
![image](https://github.com/ikhsan3adi/absensi-sekolah-qr-code/assets/22759294/9e1b884d-0063-4c00-80aa-799b78426d80)


extension di php.ini nya sudah saya aktifkan gan

**Reply by maderaiindraperkasa** [reply]
boleh tau depedencies yang perlu di install apa saja min? siapa tau saya ada yg kurang

**Reply by ikhsan3adi** [reply, marked as answer]
Harusnya semua dependencies otomatis terinstal saat menjalankan `composer install`

**Reply by maderaiindraperkasa** [reply]
done gan, saya malah install depedencies lagi manual hehee, makasih gan

**Reply by maderaiindraperkasa** [reply]
maaf gan, ternyata setelah saya ganti data kelas dan jurusan mau generate QR gak bisa
Tapi kalau menggunakan kelas dan jurusan default, dia bisa generate QR

**Reply by maderaiindraperkasa** [reply]
done gan, ternyata file img saya kegedean mungkin, maaf ya gan ehehe
